subsys/llext/Kconfig

```diff
@@ -47,6 +47,12 @@ config LLEXT_HEAP_SIZE
 	help
 	  Heap size in kilobytes available to llext for dynamic allocation
 
+config LLEXT_FILE_SYSTEM
```

Comment:
Personally I don't see the need to disable LLEXT file system support if we have support for it in the rest of Zephyr. Do you have an use case in mind? Otherwise you can directly use `FILE_SYSTEM` everywhere.

Reply:
Don't have any use case in mind. I can change the feature to be based on `FILE_SYSTEM`. If anyone will need to have extra Kconfig symbol then it will be up to her/him to submit patch